core/iwasm/common/wasm_runtime_common.h

```diff
@@ -1093,6 +1098,14 @@ wasm_runtime_quick_invoke_c_api_native(WASMModuleInstanceCommon *module_inst,
 void
 wasm_runtime_show_app_heap_corrupted_prompt();
 
+void
+wasm_munmap_linear_memory(void *mapped_mem, uint64 commit_size,
+                          uint64 map_size);
+
+void *
+wasm_mmap_linear_memory(uint64_t map_size, uint64 *io_memory_data_size,
+                        char *error_buf, uint32 error_buf_size);
+
```

Comment:
Is it better to move `wasm_munmap_linear_memory`/`wasm_mmap_linear_memory` from wasm_runtime_common.h/c to wasm_memory.h/c? We normally implement memory related operations (except shared memory) in the latter.

Reply:
Sure, done.